substreams/ethereum-lido/src/modules/3_map_protocol_changes.rs

```diff
@@ -94,9 +102,162 @@ fn handle_sync(
             };
         }
     }
+
+    // Handle storage deltas - process in chronological order to maintain correct state
+    let mut sorted_deltas = storage_deltas
+        .deltas
+        .iter()
+        .filter(|delta| {
+            delta
+                .key
+                .contains(&format!("{ST_ETH_ADDRESS_PROXY_COMPONENT_ID}:"))
+        })
+        .collect::<Vec<_>>();
+    sorted_deltas.sort_by_key(|delta| delta.ordinal);
+
+    let mut current_buffered_eth =
+        get_initial_value_for_block(&sorted_deltas, &storage_store, "buffered_eth");
+    let mut current_cl_balance_position =
+        get_initial_value_for_block(&sorted_deltas, &storage_store, "cl_balance_position");
+    let mut current_cl_validators_position =
+        get_initial_value_for_block(&sorted_deltas, &storage_store, "cl_validators_position");
+    let mut current_deposited_validators_position = get_initial_value_for_block(
+        &sorted_deltas,
+        &storage_store,
+        "deposited_validators_position",
+    );
+
+    for delta in sorted_deltas.iter() {
+        let tx_ordinal = delta.ordinal;
```

Comment:
Curious - what's an ordinal? Is this the order of the delta within the tx?

Reply:
exactly